engines/mohawk/myst.cpp

```diff
@@ -917,6 +923,17 @@ void MohawkEngine_Myst::changeToStack(MystStack stackId, uint16 card, uint16 lin
 	_cache.clear();
 	_gfx->clearCache();
 
+	// Add artificial CD-ROM delay
+	if (addCdRomDelay) {
+		if (_stack->getStackId() != kIntroStack && _stack->getStackId() != kMenuStack) {
+			// Pretty arbitrary delays to mimic a period correct 4x drive
+			// TODO: Since the disc layout of the original CD-ROMs is known,
+			//       it should be possible to adapt the delay depending on the
+			//       target stack in order to replicate the original loading times.
+			g_system->delayMillis(_rnd->getRandomNumberRng(1000, 1200));
```

Comment:
Since the number is arbitrary, you could consider removing the addCdRomDelay flag and replace it with a numeric configuration for a delay, and use that+200 random to simulate a delay. That way users can find the "correct" delay for them

Reply:
This would require switching to a slider in the GUI, or a textfield.

Reply:
Indeed, this would be a nice touch. I need to dig deeper in the GUI code to be able to do this, since I've hardly ever done any GUI code.

Maybe this could be done in a second PR?